carta/image.py

```diff
@@ -605,6 +605,189 @@ def hide_contours(self):
+        rotation_offset : {5}
+            The rotation offset in degrees. By default is 0.
+        """
+        self.call_action("vectorOverlayConfig.setThickness", thickness)
+        if intensity_min is not None and intensity_max is not None:
+            self.call_action("vectorOverlayConfig.setIntensityRange", intensity_min, intensity_max)
+        elif intensity_min is None and intensity_max is not None:
+            self.call_action("vectorOverlayConfig.setIntensityRange", "None", intensity_max)
+        elif intensity_min is not None and intensity_max is None:
+            self.call_action("vectorOverlayConfig.setIntensityRange", intensity_min, "None")
+        self.call_action("vectorOverlayConfig.setLengthRange", length_min, length_max)
+        self.call_action("vectorOverlayConfig.setRotationOffset", rotation_offset)
+
+    @validate(Color())
+    def set_vector_overlay_color(self, color):
+        """Set the vector overlay color.
+
+        This automatically disables use of the vector overlay colormap.
+
+        Parameters
+        ----------
+        color : {0}
+            The color.
+        """
+        self.call_action("vectorOverlayConfig.setColor", color)
+        self.call_action("vectorOverlayConfig.setColormapEnabled", False)
+
+    @validate(Constant(Colormap), NoneOr(Number()), NoneOr(Number()))
+    def set_vector_overlay_colormap(self, colormap, bias=None, contrast=None):
+        """Set the vector overlay colormap.
+
+        This automatically enables use of the vector overlay colormap.
+
+        Parameters
+        ----------
+        colormap : {0}
+            The colormap.
+        bias : {1}
+            The colormap bias.
+        contrast : {2}
+            The colormap contrast.
+        """
+        self.call_action("vectorOverlayConfig.setColormap", colormap)
+        self.call_action("vectorOverlayConfig.setColormapEnabled", True)
+        if bias is not None:
+            self.call_action("vectorOverlayConfig.setColormapBias", bias)
+        if contrast is not None:
+            self.call_action("vectorOverlayConfig.setColormapContrast", contrast)
+
+    def apply_vector_overlay(self):
+        """Apply the vector overlay configuration."""
+        self.call_action("applyVectorOverlay")
+
+    @validate(NoneOr(Constant(VectorOverlaySource)), NoneOr(Constant(VectorOverlaySource)), NoneOr(Number()), Boolean(), NoneOr(Number()), NoneOr(Number()), NoneOr(Number()), Number(), NoneOr(Number()), NoneOr(Number()), Number(), Number(), Number(), NoneOr(Color()), NoneOr(Constant(Colormap)), NoneOr(Number()), NoneOr(Number()))
+    def plot_vector_overlay(self, angular_source=None, intensity_source=None, pixel_averaging=4, fractional_intensity=False, threshold=None, q_error=None, u_error=None, thickness=1, intensity_min=None, intensity_max=None, length_min=0, length_max=20, rotation_offset=0, color=None, colormap=None, bias=None, contrast=None):
```

Comment:
In this function the parameters and docstring need to be updated and all the parameters also need to default to `None`, but the logic still looks correct.